app/event/send-payment-request-invalid-events.js

```diff
@@ -0,0 +1,27 @@
+const { v4: uuidv4 } = require('uuid')
+const sendPaymentRequestInvalidEvent = require('./send-payment-request-invalid-event')
+
+const sendPaymentRequestInvalidEvents = async (paymentRequests) => {
+  if (paymentRequests?.length) {
+    const events = []
+    for (const paymentRequest of paymentRequests) {
+      try {
+        events.push({
+          id: uuidv4(),
+          name: 'batch-processing-payment-request-invalid',
+          type: 'error',
+          message: `Payment request could not be processed. Error(s): ${paymentRequest.errorMessage}`,
+          data: { paymentRequest }
+        })
+      } catch {
+        console.error('Could not generate batch-processing-payment-request-invalid event for', paymentRequest)
+      }
```

Comment:
The error will still continue processing here

Reply:
The failing `paymentRequest` will not be in the `events` array and so on line 22 will not be sent